OpenEdXMobile/res/values/strings.xml

```diff
@@ -96,6 +96,8 @@
     <string name="label_close_navigation_menu">Close Navigation Menu</string>
     <!-- Button text to show user Settings screen -->
     <string name="action_settings">Settings</string>
+    <!-- Button text to show user current downloads screen -->
+    <string name="label_current_downloads" tools:ignore="MissingTranslation">View current downloads</string>
```

Comment:
Did we get approval from the docs team on this new string?

Reply:
iOS team is using "Double tap to view current downloads" just removed "Double tap to" text.

Reply:
The question is still the same. Did we get approval from docs team or not?

Reply:
Nops, didn't take any approval, in old dashboard menu we are using "Settings" which doesn't make any sense at all. For the time being i have set it as "View current downloads" and created the following story for it.

https://openedx.atlassian.net/browse/LEARNER-3278